db/migration/1709735276047-RemoveSizeDimensionFromSlopeCharts.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class RemoveSizeDimensionFromSlopeCharts1709735276047
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const slopeCharts = await queryRunner.query(
+            `SELECT id, config FROM charts WHERE type = 'SlopeChart'`
```

Comment:
Let's pull in @danyx23 here:
In some other migrations (e.g. [this one](https://github.com/owid/owid-grapher/blob/ab50cdd35c61e5b0aa4533823929979ec2869bcc/db/migration/1652693645426-RemoveMinPopulationFilter.ts)), we also apply these to the `suggested_chart_revisions` and `chart_revisions` tables.
The thinking there is that otherwise, these deleted properties will "sneak in" again in the future by way of using the suggested chart revision tool, or by recovering an old revision.

Daniel, what do you think? Should we always migrate all these tables? Is what we've been doing in the past a good pattern, or is there some better way?

Reply:
Yeah, I think this is a good pattern - in theory we now also have the etl to worry about with the grapher configs that come from there but in practice this is not relevant for this PR. So to be consistent I would say let's loop over the grapher configs in the DB and fix this everywhere it comes up. Note that by now we also have the (very often empty) grapher configs variables.grapherConfigAdmin and variables.grapherConfigETL